pyam/utils.py

```diff
@@ -268,6 +268,65 @@ def sort_data(data, cols):
     return data.sort_values(cols)[cols + ['value']].reset_index(drop=True)
 
 
+def _meta_idx(data):
+    """Return the `META_IDX` from `data` by index or columns"""
+    return data[META_IDX].drop_duplicates().set_index(META_IDX).index
+
+
+def merge_meta(left, right, ignore_meta_conflict=False):
+    """
+    Merge two ``meta`` tables
+
+    Parameters
+    ----------
+    left : pd.Dataframe
+        First meta table
+
+    right : pd.Dataframe
+        Second meta table
+
+    ignore_meta_conflict : bool, default False
+        If False, raise an error if any meta columns present in ``left`` and
+        ``right`` are not identical.
+
+    Raises
+    ------
+    ValueError
+        Values are in conflict and ``ignore_meta_conflict`` is ``False``
+
+    Returns
+    -------
+    pd.DataFrame
+        Joined metadata tables
```

Comment:
```suggestion
        Joined ``meta`` tables
```
Trying to avoid the term "metadata" to reduce confusion with actual metadata like license, source, etc.

Reply:
done